modules/sc-mesh-secure-deployment/src/1_5/common/test/spectral_capture.py

```diff
@@ -1,14 +1,35 @@
 import sys
+import os
+import pandas as pd
 sys.path.append('../')
 from SpectralMgr import Spectral
 
 if __name__ == "__main__":
+
     spec = Spectral()
     spec.initialize_scan()
     spec.execute_scan()
     f = spec.file_open("/tmp/data")
-    spec.read(f, 8512)
+    file_stats = os.stat("/tmp/data")
+    spec.read(f, file_stats.st_size)
     spec.file_close(f)
+    
+    vals_dict = []
+    vals_dict = spec.get_values()
+
+    vals_list = []
+    for key, value in vals_dict.items():
+       vals_list.append(list(value))
+
+
+    driver = os.popen('ls /sys/kernel/debug/ieee80211/phy* | grep ath').read().strip()  
```

Comment:
Move this to read function, read needs to return pd frame, later to be used by ML class